kernel/include/timeout_q.h

```diff
@@ -22,13 +22,19 @@ extern "C" {
 
 static inline void _init_timeout(struct _timeout *t, _timeout_func_t fn)
 {
-	t->dticks = _INACTIVE;
+	sys_dnode_init(&t->node);
+	t->dticks = 0;
 }
 
 void _add_timeout(struct _timeout *to, _timeout_func_t fn, s32_t ticks);
 
 int _abort_timeout(struct _timeout *to);
 
+static inline bool _is_inactive_timeout(struct _timeout *t)
```

Comment:
@andyross The sense of this can be reversed to `_is_active_timeout()` if #12264 is resolved.  I might be able to include a fix for that if you still want this change made.